### PR TITLE
build(dev): bump ruff version

### PR DESCRIPTION
**Pull request by afuetterer**
## Description

This PR bumps the ruff version in pre-commit configuration. This will be done by pre-commit.ci as well, but it will not fix the "ruff (legacy alias)" warning, which was introduced by ruff. ruff now is called by the id: "`ruff-check`".

Ref: https://github.com/astral-sh/ruff-pre-commit/blob/1b111be1f83c4bb53b82476ca4562714ff2b0612/.pre-commit-hooks.yaml#L1

### CODE REVIEW

**Comment by MyPyDavid** [comment]
yes, just like this `_, b = foo()`

**Comment by afuetterer** [comment]
Ref: https://docs.astral.sh/ruff/rules/unused-unpacked-variable/

**Comment by afuetterer** [comment]
I don't care at all, as long as it is consistent. The easiest fix and not conflicting with django lazytext stuff would be just to accept the autofix in my opinion (e.g. `_elements`). But yes, you can discuss.

I overlooked that @MyPyDavid claimed this PR. So its your call then.

**Comment by jochenklar** [comment]
There is an agreement! `_x` ist is.

File: rdmo/projects/tests/test_command_sync_projects.py [hunk 1]
```diff
@@ -90,8 +90,8 @@ def test_command_sync_projects_for_tasks_and_views_with_show(
     assert settings.PROJECT_VIEWS_SYNC
 
     # Arrange task and view state
-    P1, C1, T = arrange_projects_catalogs_and_tasks()
-    P2, C2, V = arrange_projects_catalogs_and_views()
+    P1, _C1, T = arrange_projects_catalogs_and_tasks()
```
Comment (afuetterer) [inline rdmo/projects/tests/test_command_sync_projects.py]: Not using _x here, because it is already defined earlier? Edge case of your _x rule?

Comment (MyPyDavid) [reply rdmo/projects/tests/test_command_sync_projects.py]: after the `--unsafe-fixes` I wanted to replace everything with the `_x` and I probably just missed that one :/ . I thought before that maybe it doesn't need to be so strict in the tests but on the other hand it may be easier to keep it(this unconventional code style compromise) consistent throughout.
